spec/requests/invites_spec.rb

```diff
@@ -50,6 +50,44 @@
         subject
       end
     end
+
+    describe "POST /api/invites/:id/resend" do
+      let(:mail) { double("Mail::Message") }
+
+      subject { post "/api/invites/#{invite.id}/resend" }
+
+      it "sends a message" do
+        expect(OnboardingMailer).to receive(:welcome_email).and_return(mail)
+        expect(mail).to receive(:deliver_later)
+
+        subject
+        expect(response).to be_no_content
+      end
+
+      it "404s when the invite has been claimed" do
+        invite.redeemer = student
```

Comment:
To replace the assignment and `save!`, we could use `invite.update(redeemer: student)`.